alt_requirements/requirements_pretrained_embeddings_convert.txt

```
@@ -1,5 +1,5 @@
 # Minimum Install Requirements
 -r ../requirements.txt
 
-tensorflow_text==1.15.1
-tensorflow_hub==0.6.0
+tensorflow_text>=2.1.0rc0
```

@Ghostvv please also pin this one in this file (setup.py is fine) (or to 2.1.0)
```suggestion
tensorflow_text==2.1.0rc0
```

---

@tmbo should we pin it though? there was a problem with installing `tensorflow_text==2.1.1`, hence this work around